pandas/tests/reshape/concat/test_series.py

```diff
@@ -120,24 +120,26 @@ def test_concat_series_name_npscalar_tuple(self, s1name, s2name):
 
     def test_concat_series_partial_columns_names(self):
         # GH10698
-        foo = Series([1, 2], name="foo")
-        bar = Series([1, 2])
-        baz = Series([4, 5])
+        foo_series = Series([1, 2], name="foo")
+        bar_series = Series([1, 2])
+        baz_series = Series([4, 5])
```

Comment:
If going through the trouble to rename, probably worth also improving the`foo/bar/baz` naming. Otherwise we're just changing the variable name solely to please `pylint` not for code quality. Maybe something like `named`, `unnamed1`, `unnamed2`?

Reply:
thx, good suggestions